package.json

```diff
@@ -15,6 +15,7 @@
   "scripts": {
     "start": "npx nodemon server.js",
     "test": "jest --env=jsdom --watchAll",
+    "testCoverage": "jest --env=jsdom --coverage",
```

Comment:
This is a necessary and important addition to our test functionality. Great job!